src/lib/OS/SyscallHandler.cc

```diff
@@ -1328,14 +1482,21 @@ int64_t SyscallHandler::schedSetAffinity(pid_t pid, size_t cpusetsize,
   // Currently, the bit mask can only be 1 so capture any error which would
   // occur but otherwise omit functionality
   if (mask == 0) return -EFAULT;
+  uint64_t translatedAddr =
+      OS_->handleVAddrTranslation(mask, currentInfo_.threadId);
+  uint64_t faultCode = simeng::OS::masks::faults::getFaultCode(translatedAddr);
+  if (faultCode == simeng::OS::masks::faults::pagetable::DATA_ABORT ||
+      faultCode == simeng::OS::masks::faults::pagetable::IGNORED) {
+    return -EFAULT;
```

Comment:
Is this value consistent across macOS and Linux, if it is please resolve this comment. Thanks

Reply:
MacOS and Linux values match (both 14)

Reply:
Do you think these should be moved to Constants.hh for consistency?

Reply:
I don't think so. May cause confusion as the libary defined values will work